### DISCUSSION TITLE
array_merge() Error when use validation

### DISCUSSION THREAD

**Discussion by yanalshoubaki** [livewire/livewire#8230] [7. Bugs (archived)] [answered]
### Livewire version

v3.4.9

### Laravel version

v11.1.1

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

1. create a new livewire component
2. add property name $messages 
3. use a validation method to validate the form

### Repository or Wirebox URL

https://wirebox.app/b/4m8jr

### Additional information

I know maybe we can't use this variable $messages because it handles messages, but I guess we can add a type check for it so that anyone can use the correct variable for his component to be more clearable when anyone read his code

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@yanalshoubaki yeah `$messages` is used internally by Livewire so you can't use them. PRs to the docs to make that clearer are welcome!

**Comment by PerryvanderMeer** [comment, marked as answer]
See https://github.com/livewire/livewire/pull/8254

**Reply by yanalshoubaki** [reply]
Thank you for updating the docs to be more clarification for the developer, I will close the discussion 🔥
